core/templates/pages/exploration-editor-page/editor-navigation/editor-navbar-breadcrumb.component.ts

```diff
@@ -65,7 +65,7 @@ angular.module('oppia').component('editorNavbarBreadcrumb', {
       ctrl.$onInit = function() {
         $scope.navbarTitle = null;
         ctrl.directiveSubscriptions.add(
-          ExplorationPropertyService.onExplorationPropertyChanged.subscribe(
+          ExplorationTitleService.onExplorationPropertyChanged.subscribe(
             (propertyName) => {
               if (propertyName === ExplorationTitleService.propertyName) {
                 const _MAX_TITLE_LENGTH = 20;
```

Comment:
If we make this change, then we should no longer need the `if` statement on line 70.

Reply:
DONE

Reply:
Also, why did you decide to make this change in the first place? Why not keep it as `ExplorationPropertyService`? @heyimShivam

Reply:
I thought if we inherited explorationPropertyService in ExplorationTitleService then we don't need ExplorationPropertyService to be used differently.` Why not keep it as ExplorationPropertyService?   Done ` @EricZLou PTAL

Reply:
after doing this changes one e2e fails

Reply:
@EricZLou

Reply:
https://user-images.githubusercontent.com/77006897/147371429-56ef6bd9-5cec-4684-a378-5e6ca89a6fa5.mp4

Reply:
used `ExplorationTitleService`  instead of ExplorationPropertyService